inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/pooling/pooling_kernel_gpu_ref.cpp

```diff
@@ -27,6 +27,7 @@ ParamsKey PoolingKernelGPURef::GetSupportedKey() const {
     k.EnablePoolKernelDividerMode(KernelDividerMode::DYNAMIC);
     k.EnablePoolKernelDividerMode(KernelDividerMode::DYNAMIC_WITH_PADDING);
     k.EnableDifferentTypes();
+    k.EnableMaxPoolOpset8Features();
```

Comment:
MaxPool-8 has 2 additional features comparing to MaxPool-1. EnableMaxPoolOpset8Features key can be split into 2 keys:
* EnableDilations
* EnableOutputSelectedIndeces